opencti-platform/opencti-front/src/utils/Time.js

```diff
@@ -82,6 +82,12 @@ export const dateFormat = (data, specificFormat = null) => {
     : '';
 };
 
+export const validStartTimeForRelationCreation = (time) => {
```

Comment:
It would be better to have a generic function allowing to subtract `x` seconds from a date instead of a very specific function. You have an example in this same file with function `minutesBefore`

Reply:
As we could remove 1 minute (instead of 1 seconde as I mentioned above), I used `minutesBefore` that was already existing and I deleted `validStartTimeForRelationCreation `